Source/Extensions/UIViewExtensions.swift

```diff
@@ -68,3 +68,72 @@ extension UIView {
   }
 
 }
+
+extension UIView {
+
+    func addBorder(with list: [SideBorder], thickness: Double, color: UIColor) {
+        list.forEach {
+            addBorder(side: $0, thickness: thickness, color: color)
```

Comment:
Code level I'm not sure this approach will work if you have right and bottom border with corner radius.